packages/google_maps_flutter/google_maps_flutter_web/lib/src/circle.dart

```diff
@@ -26,21 +26,23 @@ class CircleController {
 
   /// Returns the wrapped [gmaps.Circle]. Only used for testing.
   @visibleForTesting
-  gmaps.Circle get circle => _circle;
+  gmaps.Circle? get circle => _circle;
 
   /// Returns `true` if this Controller will use its own `onTap` handler to consume events.
   bool get consumeTapEvents => _consumeTapEvents;
 
   /// Updates the options of the wrapped [gmaps.Circle] object.
   void update(gmaps.CircleOptions options) {
-    _circle.options = options;
+    _circle!.options = options;
```

Comment:
Either this should have a null check, or the docs should say that it's a programming error to call it after `remove()`.

Reply:
I've added an assertion, and extra docs to specify what methods can't be called after `remove`. I'll add some tests too.